src/docfx/serve/lsp/credentialRefresh/CredentialRefreshParams.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using MediatR;
+using OmniSharp.Extensions.JsonRpc;
+
+namespace Microsoft.Docs.Build
+{
+    [Parallel]
+    [Method("docfx/userCredentialRefresh", Direction.ServerToClient)]
```

Comment:
consider use a verb (as other lsp server to client protocols like `registerCapability`), and also the term `refresh` is a bit limited because we could theoretically use this protocol to retrieve the credentials for the first time.
```suggestion
    [Method("docfx/getCredential", Direction.ServerToClient)]
```